contrib/hadoop-store-builder/src/java/voldemort/store/readonly/fetcher/HdfsFetcher.java

```diff
@@ -210,6 +210,36 @@ public File fetch(String sourceFileUrl,
         }
     }
 
+    /**
+     * Replace swebhdfs protocol and port in sourceFileUrl when SSL is not enabled in this node.
+     */
+    protected String modifyURLbyConfig(String sourceFileUrl) {
```

Comment:
Do you think it'd make sense to move this code to a Utils class, and take newProtocol and newPort as input parameter?

This way, we could call the same code from HdfsFetcher as well as from the ```VoldemortSwapJob``` (here: https://github.com/voldemort/voldemort/blob/6050ab929a807aa32397ce2aee752320cde9d763/contrib/hadoop-store-builder/src/java/voldemort/store/readonly/mr/azkaban/VoldemortSwapJob.java#L97-L113).